extensions/wikia/AssetsManager/config.php

```diff
@@ -2448,7 +2448,8 @@
 	'type' => AssetsManager::TYPE_SCSS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
-		'//extensions/wikia/PortableInfobox/styles/PortableInfobox.scss'
+		'//extensions/wikia/PortableInfobox/styles/PortableInfobox.scss',
+		'//extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss'
```

Comment:
This will result in multiple versions on Varnish; not only combinations of assets coming from multiple articles, but also from the contents of the article. From the cache perspective (and UX), based on the size of this, I'd actually leave it as-is.